.evergreen.yml

```diff
@@ -78,6 +78,7 @@ functions:
         env:
           NODE_JS_VERSION: ${node_js_version}
           DISTRO_ID: ${distro_id}
+          MONOGDB_VERSION_OVERRIDE: ${mongodb_version_override}
```

Comment:
It's pretty unobvious that this refers to `mongodb` as in, the name of the literal `mongodb` driver npm package. Maybe `mongodb_driver_version_override` would be less ambiguous?